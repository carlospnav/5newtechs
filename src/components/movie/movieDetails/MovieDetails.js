import styled from "styled-components"
import { flexGroup } from "theme/utils"

export default styled.section`
  ${flexGroup(null, "center")};
  flex-wrap: wrap;

  @media (min-width: 1000px){
    justify-content: space-between;
  }
`