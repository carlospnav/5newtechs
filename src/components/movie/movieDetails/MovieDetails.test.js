import { shallow } from "enzyme"
import MovieDetails from "./MovieDetails"
import "jest-styled-components"

describe("<MovieDetails>", () => {
  it("should render a MovieDetails properly.", () => {
    const wrapper = shallow(<MovieDetails />)
    expect(wrapper).toMatchSnapshot()
  })
})