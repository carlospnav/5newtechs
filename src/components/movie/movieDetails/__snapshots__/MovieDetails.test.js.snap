// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieDetails> should render a MovieDetails properly. 1`] = `
.c0 {
  -webkit-flex-direction: null;
  -ms-flex-direction: null;
  flex-direction: null;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

@media (min-width:1000px) {
  .c0 {
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    -ms-flex-pack: justify;
    justify-content: space-between;
  }
}

<section
  className="c0"
/>
`;
